content/en/docs/tasks/administer-cluster/access-cluster-api.md

```diff
@@ -351,103 +351,3 @@ exampleWithKubeConfig = do
             (CoreV1.listPodForAllNamespaces (Accept MimeJSON))
         >>= print
 ```
-
```

Comment:
To preserve existing links to the section and let readers find the new task page, add something like this at the end of the page:
```markdown

## {{% heading "whatsnext" %}}


- <span id="accessing-the-api-from-within-a-pod" />
  Read about [Accessing the API from within a Pod](/docs/tasks/access-api-from-pod/access-api-from-pod/)
```

Reply:
Added